Makefile

```diff
@@ -443,6 +445,10 @@ site/src/api/typesGenerated.ts: scripts/apitypings/main.go $(shell find codersdk
 	cd site
 	yarn run format:types
 
+cli/testdata/golden: $(wildcard cli/testdata/*.golden)
```

Comment:
The job should be something like `cli/testdata/.generated`, then you can `touch "$@"` at the bottom of your task and remove the `.PHONY`

PHONY jobs get run all the time if they are requested, so the mark-fresh stuff wouldn't be necessary if you did want this to be PHONY.

Reply:
Nice, thanks! I was sure there was a better way of doing it and ofc you knew! 😄

Reply:
Hmm, I noticed that this will create a file named `cli/testdata/.generated`. I guess we should gitignore it? But then, how should we do this when someone adds a new test case and we need a new `.golden` file to be generated? Simply running `make gen` would not produce any output.

Reply:
You should add dependencies on all go files in that case as well. Check how we do it on another make task using `find`.